src/test/lumo/lumo/build_api_tests.cljs

```diff
@@ -376,12 +376,39 @@
     (testing ":optimizations :none"
       (test/delete-out-files out)
       (is (nil? (build/build
-                  (path/join root "test_check")
-                  {:main 'hello
-                   :optimizations :none
-                   :output-to out}
-                  (env/default-compiler-env))) "It should successfully compile with :optimizations :none")
-        (test/delete-out-files out))))
+                 (path/join root "test_check")
+                 {:main 'test-check.summary.core
```

Comment:
why did you change this test?

Reply:
Sorry it is not changed I will revert...probably some copy and paste in the wrong place